webui/react/src/components/CreateGroupModal.tsx

```diff
@@ -192,11 +194,13 @@ const CreateGroupModalComponent: React.FC<Props> = ({ onClose, users, group }: P
           </Form.Item>
           <Form.Item label={USERS_LABEL} name={USERS_NAME}>
             <Select mode="multiple" optionFilterProp="children" placeholder="Add Users" showSearch>
-              {users?.map((u) => (
-                <Select.Option key={u.id} value={u.id}>
-                  {getDisplayName(u)}
-                </Select.Option>
-              ))}
+              {users
+                ?.filter((u) => u.isActive || currentGroupMembers?.includes(u.id))
```

Comment:
**nit suggestion:** our modals seem to re-render a lot, where having this filters user list cached (via `useMemo`) could help the performance slightly.

Reply:
I tried memo-izing `currentGroupMembers` based on `[form]` and it didn't work, so for now I'll keep it as-is